yurt/fabfile/setup.py

```diff
@@ -1,73 +1,73 @@
 import os
 import click
-from fabric.context_managers import settings, lcd
-from fabric.contrib.files import append
-from fabric.operations import local, run
-from fabric.state import env
-from utils import recursive_file_modify, get_fab_settings, \
-    generate_ssh_keypair, get_project_name_from_repo
+from invoke import run
+from utils import recursive_file_modify, \
+    generate_ssh_keypair, get_project_name_from_repo, add_settings
 from cli import main
 
 __author__ = 'deanmercado'
 
 YURT_PATH = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
 DJANGO_PROJECT_PATH = os.path.join(YURT_PATH, 'django_project')
 ORCHESTRATION_PROJECT_PATH = os.path.join(YURT_PATH, 'orchestration')
+FABFILE_PATH = os.path.join(YURT_PATH, 'fabfile')
 
 
-def create_settings():
-    fabric_settings = get_fab_settings()
+def create_settings(vault, git_repo):
+    fabric_settings = add_settings(vault, git_repo)
```

Comment:
@rmutter So `fabric_settings` is a misnomer. It's a dictionary kept in memory and used to populate values in the templates when generating the codebase. I should rename that since we're no longer making the user create a `fabric_settings` file.